blocks/library/image/style.scss

```diff
@@ -17,9 +17,14 @@
 	}
 
 	&[data-align="wide"] {
+		margin-left: -24%;
```

Comment:
Hmm yes this, for some reason, doesn't actually scale to the viewport. Replacing it with `5vw` is seems to behave as intended.